autoload/verilog_systemverilog.vim

```diff
@@ -587,6 +587,11 @@ function! verilog_systemverilog#VerilogErrorFormat(...)
   if (l:tool == "ncverilog")
     " Based on https://github.com/vhda/verilog_systemverilog.vim/issues/88
     set errorformat=%.%#:\ \*%t\\,%.%#\ \(%f\\,%l\|%c\):%m
+    set errorformat+=%.%#\ *%t\\,%.%#(%f\\,%l\|%c):\ %m
+    " Assertion notices don't have column information
+    set errorformat+=%.%#\ *%t\\,%.%#(%f\\,%l):\ %m
```

Comment:
For the following format:
ncsim: *E,ASRTST (filename.sv,769): (time 0 FS) Assertion assertion_name has failed